dfget/core/core.go

```diff
@@ -175,11 +177,10 @@ func downloadFile(cfg *config.Config, supernodeAPI api.SupernodeAPI,
 	}
 
 	timeout := calculateTimeout(cfg.RV.FileLength, cfg.Timeout, cfg.MinRate)
-	err := downloader.DoDownloadTimeout(getter, timeout)
-	success := "SUCCESS"
-	if err != nil {
+	success := true
+	if err = downloader.DoDownloadTimeout(getter, timeout); err != nil {
 		logrus.Error(err)
```

Comment:
same unreadable log here. Please help to update this.Thanks

Reply:
I prefer removing this log here. Because we will log the error and download status in the end of the function. Maybe we can add more fields in that log.
``` Go
	if success {
		logrus.Infof("download SUCCESS cost:%.3fs length:%d",
			time.Since(cfg.StartTime).Seconds(), cfg.RV.FileLength)
	} else {
		logrus.Infof("download FAIL cost:%.3fs length:%d reason:%d",
			time.Since(cfg.StartTime).Seconds(), cfg.RV.FileLength, cfg.BackSourceReason)
	}
```